tools/autograd/derivatives.yaml

```diff
@@ -1791,8 +1800,6 @@
 
 - name: glu(Tensor self, int dim=-1) -> Tensor
   self: glu_backward(grad, self, dim)
-  # RuntimeError: output with shape [1] doesn't match the broadcast shape [2]
```

Comment:
Should this be added back?

Reply:
Nope. I realized that this op isn't even element-wise... it actually splits the input in half and then applies binary-element-wise, so this error is expected if we attempt to do auto-element-wise